PxCs/Interface/PxSound.cs

```diff
@@ -0,0 +1,165 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+using PxCs.Extensions;
+
+
+namespace PxCs.Interface
+{
+    public class PxSound
+    {
+        private const string DLLNAME = PxPhoenix.DLLNAME;
+
+        public static byte[] GetSoundByteArrayFromVDF(IntPtr vdfPtr, string name)
```

Comment:
This is an uppercased method. Can you please return a PxData object. Look inside other Interface classes. This one should follow similarly please.

E.g. this is how Mrm is fetched from PxCs: https://github.com/GothicVRProject/GothicVR/blob/feature/audio-implementation/Assets/GothicVR/Scripts/Caches/AssetCache.cs#L119